docs/Corbnorb/noise.md

```diff
@@ -0,0 +1,70 @@
+# FastNoiseLite
+
+## Creator
+I did not create the noise, I only turned it into an extension.
+For the original project go here:
+https://github.com/Auburn/FastNoiseLite
+https://auburn.github.io/FastNoiseLite/
+
+## How to use
+
+All features you can test on this website: https://auburn.github.io/FastNoiseLite/
+*except for easing and inverted*
+
+There are 2 blocks:
+
+1. [create noise](#createNoise)
+2. [get noise](#getNoise)
+
+Although the [get noise](#getNoise) block has 3D coordinates, you can use just X and Y if you want 2D noise
+
+### Create Noise <a name="createNoise"></a>
```

Comment:
Please don't use HTML elements in the documentation.

Reply:
No this is fine